test/integration/test_integration.py

```diff
@@ -177,6 +178,9 @@ def results(request):
 _pycallgraph = fixtures_factory('python-pycallgraph-0.5.1-13.fc28')
 pycallgraph = fixtures_factory('_pycallgraph')
 
+_ttystatus = fixtures_factory('python-ttystatus-0.34-7.fc29')
+ttystatus = fixtures_factory('_ttystatus')
```

Comment:
Note that I'd rather replace pycallgraph with another package, but i couldn't find anything that would fit for both usecases  pycallgraph had.